src/Http/Controllers/CartItemController.php

```diff
@@ -16,11 +16,21 @@ public function store(Request $request)
             'product_type' => 'required|string',
             'product_id' => 'required',
             'quantity' => 'nullable|numeric',
-            'options' => 'nullable|array'
+            'options' => 'nullable|array',
+            'multi' => 'nullable|string'
```

Comment:
Shouldn't it be a `boolean`?
```php
            'multi' => 'nullable|boolean'
```

Reply:
basically itt shouldn't be at all, but i didn't know how you used this package in other projects, or how you will want to use it in future, so i did it like this